### ISSUE TITLE
Participatory Modeling Planning

### ISSUE THREAD

**Issue by ththelen** [NCSU-CHAZ/Sunny-Day-Flooding#23]
Forum for TT and @anardek to coordinate on CB participatory modeling. TT PM lit review and proposal attached to this description for background.

[AnnotatedBibliography.docx](https://github.com/NCSU-CHAZ/Sunny-Day-Flooding/files/10279720/AnnotatedBibliography.docx)

**Comment by ththelen** [comment]
_@anardek feedback on the annotated bibliography:_ Things that I think are worth discussing moving forward: 1) what is the timeline, 2) do the participants get paid?, 3) will we need to hire someone to run the events? I think it would be good to come up with a series of logistical questions for our meeting in January.

**Comment by ththelen** [comment]
**Timeline:** Since we will be using contacts identified through Miyuki’s survey outreach, we cannot start any PM until after those contacts are established. However, we should run our plan by the town before reaching out to folks. I have listed a potential order of operations below.
- Draft inquiry about interest in participating in PM to include in Miyuki’s survey.
- Coordinate with town on PM.
- MH conducts survey. Team reviews survey results for PM candidates.
- Reach out to PM candidates. Assemble working group. 
- Schedule 1st pm meeting. Meetings outline below
o	_Meeting 1:_ Explain the model, how it works. Field questions. Get all participants up to speed on the model. End with quick reactions on things the model might be missing.
o	_Meeting 2:_ Get more feedback on the model. Rebuild where possible. Brainstorm hindcast flood events to simulate.
o	_Meeting 3:_ Share results from hindcast flood simulations. Check match against reality. Brainstorm adaptation measures, future SLR scenarios to simulate.
o	_Meeting 4_: Share future scenarios simulation results. Get feedback on PM process. Ask questions to see if stakeholder perceptions of flood risk and flood adaptation measures (especially retreat) change after involvement in participatory modeling.

**Participant compensation:** None of the literature I have read has specified how much participants were paid. The Hemmerling work I reviewed did provide honorariums, while a focus group run by LA-Lafayette researchers to assess participant community understandings of flood risk and mitigation did not provide compensation. A useful recommendation I heard at AGU was to consider what folks are giving up to participate in research and compensate accordingly. Examples included providing meals or childcare. I think it would be worth reaching out to Hemmerling et al. to see what they provided as compensation. Also, we should have pizza at the PM meetings.

**Hiring PM coordinator/helper:** I had not considered having someone else run the PM sessions. I think the only way it makes sense is if we hire someone to lead the sessions who already has connections in Carolina Beach that might allow them to connect with stakeholders in a way that I cannot. While I had envisioned leading the session myself, more helping hands to facilitate could definitely be useful. Based on what I know about Ryan’s interests, I suspect he might be interested in helping out. If we wanted to look at bringing on an undergrad, I would suggest recruiting someone who is outside of engineering. An anthropologist/social science background student with formal training on engaging with stakeholder could bring a useful background that I am currently lacking. 
We should discuss in greater detail what sources of funding are available for this work. With participant compensation, potentially brining on helper, and other expenses (e.g., travel), I suspect it will not be a trivial investment. The NC WRRI had a Community Collaborative Research Grant Program funding opportunity with awards between $5,000 and $25,000 that would have been a great fit for this work, but the request for proposals closed Dec 21. https://wrri.ncsu.edu/wp-content/uploads/2022/10/CCRG-RFP-2022-23_Final.pdf
